community/kernel/src/main/java/org/neo4j/helpers/Service.java

```diff
@@ -301,10 +301,7 @@ protected T fetchNextOrNull()
                             }
                             catch ( Throwable e )
                             {
-                                if ( printServiceLoaderStackTraces )
-                                {
-                                    e.printStackTrace();
-                                }
+                                e.printStackTrace();
```

Comment:
Yeah, the right thing to do is to pass in our Logging here, but `Service` is a global utility, so it needs to be refactored. I modified this to at least always print the error because writing this code is the fourth or fifth time I've lost hours debugging KernelExtensions because they don't print exceptions when they fail to load. Eg: I think this is an improvement, even if it doesn't fully solve the problem